<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SailfishWizards::Internal::DBusInterfaceSnippetPage</class>
 <widget class="QWidget" name="SailfishWizards::Internal::DBusInterfaceSnippetPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>485</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Insert DBusInterface</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="busType">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Bus type:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="changeBusType">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="serviceName">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Service selection:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="changeService">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="servicePath">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Path:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="changeServicePath">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="interfaceName">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Interface selection:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="changeInterface">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Methods:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableView" name="methodsTable">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select the methods you are want to use. Selecting a method from the list will filter the signals and methods defined in the same interface as the selected one.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <attribute name="horizontalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Signals:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableView" name="signalsTable">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select the signals you want to use. Selecting a signal from the list will filter the signals and methods defined in the same interface as the selected one.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <attribute name="horizontalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="advancedMode">
           <property name="text">
            <string>Advanced mode</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="cppFileFlag">
           <property name="text">
            <string>CPP file</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="qmlFileFlag">
           <property name="text">
            <string>QML file</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="errorLabel">
         <property name="palette">
          <palette>
           <active>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>239</red>
               <green>41</green>
               <blue>41</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>239</red>
               <green>41</green>
               <blue>41</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>190</red>
               <green>190</green>
               <blue>190</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
